docs/src/mdx-components.tsx

```diff
@@ -92,6 +92,7 @@ export const td = ({
   <TableCell
     as="td"
     align={align === "left" || align === "center" || align === "right" ? align : undefined}
+    whiteSpace="normal"
```

Comment:
The tables where horizontally overflowing in `DocLayout.tsx` because table cells by default don't wrap, not sure why.